utils/generate_sample_data.py

```diff
@@ -0,0 +1,192 @@
+# Copyright 2025 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Standalone script to generate dummy data for Timesketch."""
+
+import argparse
+import csv
+import datetime
+import random
+import string
+import sys
+from datetime import timezone
+
+from tqdm import tqdm
```

Comment:
Is it really required for this script? I mean, yes it is nice etc. But it is the only dependency that you have to install separately. Not using it would make this script work with a default python install I think.

Reply:
ACK, removed